### PR TITLE
Update to Extreme Tux Racer 0.8.2

### PR DESCRIPTION
**Pull request by Newbytee**
I tried using upstream's metainfo, but it has issues (doesn't validate and is installed as etr.appdata.xml rather than etr.metainfo.xml) so I'm keeping the downstream one for now.

### CODE REVIEW

**Comment by hfiguiere** [comment]
> I tried using upstream's metainfo, but it has issues (doesn't validate

Patch then

> and is installed as etr.appdata.xml rather than etr.metainfo.xml) so I'm keeping the downstream one for now.

this actually doesn't matter.

**Comment by Newbytee** [comment]
bot, build

File: net.sourceforge.ExtremeTuxRacer.json [hunk 1]
```diff
@@ -41,17 +41,16 @@
...
-                    "type": "file",
-                    "path": "etr.metainfo.xml"
+                    "type": "patch",
+                    "path": "fix-metainfo.patch",
+                    "strip-components": 0
                 }
             ],
             "post-install": [
-                "mkdir -p /app/share/metainfo",
-                "install -Dm644 etr.metainfo.xml /app/share/metainfo",
```
Comment (hfiguiere) [inline net.sourceforge.ExtremeTuxRacer.json:54]: you probably need to add a `rename-appdata-file` in the top level to rename the upstream metainfo.

File: etr.metainfo.xml [hunk 2]
```diff
@@ -37,6 +37,7 @@
     </screenshot>
```
Comment (hfiguiere) [inline etr.metainfo.xml]: and since you patch upstream now, you should remove this file from the repo.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by Newbytee** [comment]
bot, build

**Review by hfiguiere — APPROVED** [review]

**Pull request merged**